### PR TITLE
Login modal

### PR DESCRIPTION
**Pull request by AkshatPandey1**
# Features
- Login Modal 
- Displays title, Error message with a close button and a cross button
- Connected to backend message replies
image

### CODE REVIEW

File: frontend/src/components/Login/Login.jsx [hunk 1]
```diff
@@ -29,6 +30,10 @@ const textContent = {
 
 function Login(props) {
   const content = textContent[props.type];
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
```
Comment (Manpreet-Singh001) [inline frontend/src/components/Login/Login.jsx:35]: Do we need to assign this? isn't setShow already doing this.

File: frontend/src/components/Login/Login.jsx [hunk 2]
```diff
@@ -43,23 +48,31 @@ function Login(props) {
         .then((response) => {
```
Comment (Manpreet-Singh001) [inline frontend/src/components/Login/Login.jsx:48]: nit: I prefer await syntax.

File: frontend/src/components/Login/Login.jsx (hunk 1 above)
Comment (AkshatPandey1) [reply frontend/src/components/Login/Login.jsx:35]: Ah, I just copied that code from Bootstrap's Modal docs, we can remove the functions if required

File: frontend/src/components/Login/Login.jsx (hunk 2 above)
Comment (AkshatPandey1) [reply frontend/src/components/Login/Login.jsx:48]: Sure, we can go for the await syntax. Do you think we should leave it like that in Login/Register but apply the await syntax in the rest of the code?

**Review by Manpreet-Singh001 — APPROVED** [review]

**Pull request merged**